metricbeat/module/kubernetes/apiserver/_meta/fields.yml

```diff
@@ -36,8 +36,11 @@
       type: long
       description: >
         Request latency, number of requests
-    - name: request.latency.bucket.*
```

Comment:
bucket.* does not exist anymore? If yes, breaking change?

Reply:
I'll remove the apiserver code, this was mostly a way of testing for existing prometheus metrics.

yes, it would be a breaking change for any existing prometheus metrics endpoint, but let's save it for another PR